static/app/components/events/aiAutofix/useAiAutofix.tsx

```diff
@@ -12,9 +12,9 @@ const POLL_INTERVAL = 2500;
 export const useAiAutofix = (group: GroupWithAutofix) => {
   const api = useApi();
 
-  const [additionalContext, setAdditionalContext] = useState('');
-  const [overwriteData, setOverwriteData] = useState<AutofixData | null>(null);
-  const autofixData = overwriteData ?? group.metadata?.autofix ?? null;
+  const [overwriteData, setOverwriteData] = useState<AutofixData | 'reset' | null>(null);
```

Comment:
nit - kinda weird type to mash together

Reply:
Yeah not a huge fan either, I'll probably refactor this at some point to not have an overwrite data state. Still iterating on this a bunch though so don't think it's worth a better solution yet.